gdk4/src/subclass/content_provider.rs

```diff
@@ -0,0 +1,445 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{Clipboard, ContentFormats, ContentProvider};
+
+use glib::object::IsA;
+use glib::subclass::prelude::*;
```

Comment:
```suggestion
use crate::subclass::prelude::*;
```
to be consistent with the changes landed at #323

Reply:
ok